accounts/instrument_details.go

```diff
@@ -0,0 +1,68 @@
+package accounts
+
+import (
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/instruments"
+)
+
+type InstrumentDetail string
+
+type (
+	InstrumentDetails interface {
+		GetType() string
+	}
+
+	InstrumentDetailsFasterPayments struct {
+		AccountNumber string `json:"account_number,omitempty"`
+		BankCode      string `json:"bank_code,omitempty"`
+	}
+
+	InstrumentDetailsSepa struct {
+		Iban     string `json:"iban,omitempty"`
+		SwiftBic string `json:"swift_bic,omitempty"`
+	}
+)
+
+type (
+	PaymentInstrumentDetailsResponse struct {
+		HttpMetadata       common.HttpMetadata        `json:"http_metadata,omitempty"`
+		Id                 string                     `json:"id,omitempty"`
+		Status             InstrumentStatus           `json:"status,omitempty"`
+		InstrumentId       string                     `json:"instrument_id,omitempty"`
+		Label              string                     `json:"label,omitempty"`
+		Type               instruments.InstrumentType `json:"type,omitempty"`
+		Currency           common.Currency            `json:"currency,omitempty"`
+		Country            common.Country             `json:"country,omitempty"`
+		DefaultDestination bool                       `json:"default,omitempty"`
+		Document           *InstrumentDocument        `json:"document"`
+	}
+
+	PaymentInstrumentQueryResponse struct {
+		HttpMetadata common.HttpMetadata                `json:"http_metadata,omitempty"`
+		Data         []PaymentInstrumentDetailsResponse `json:"data,omitempty"`
+	}
+)
+
+type (
+	PaymentInstrumentRequest struct {
+		Label              string                     `json:"label,omitempty"`
+		Type               instruments.InstrumentType `json:"type,omitempty"`
+		Currency           common.Currency            `json:"currency,omitempty"`
+		Country            common.Country             `json:"country,omitempty"`
+		DefaultDestination bool                       `json:"default,omitempty"`
+		Document           *InstrumentDocument        `json:"document"`
+		InstrumentDetails  InstrumentDetails          `json:"instrument_details,omitempty"`
+	}
+
+	PaymentInstrumentsQuery struct {
+		Status InstrumentStatus `json:"status,omitempty"`
+	}
+)
+
+func (s InstrumentDetailsFasterPayments) GetType() string {
```

Comment:
This should be a pointer receiver: 
`func (s *InstrumentDetailsFasterPayments) GetType() string ...`

Reply:
👍